cub/device/dispatch/dispatch_adjacent_difference.cuh

```diff
@@ -117,27 +116,16 @@ struct DeviceAdjacentDifferencePolicy
   // Architecture-specific tuning policies
   //------------------------------------------------------------------------------
 
-  struct Policy300 : ChainedPolicy<300, Policy300, Policy300>
-  {
-    using AdjacentDifferencePolicy =
-      AgentAdjacentDifferencePolicy<128,
-                                    Nominal8BItemsToItems<ValueT>(7),
-                                    BLOCK_LOAD_WARP_TRANSPOSE,
-                                    LOAD_DEFAULT,
-                                    BLOCK_STORE_WARP_TRANSPOSE>;
-  };
-
-  struct Policy350 : ChainedPolicy<350, Policy350, Policy300>
-  {
-    using AdjacentDifferencePolicy =
-      AgentAdjacentDifferencePolicy<128,
-                                    Nominal8BItemsToItems<ValueT>(7),
-                                    BLOCK_LOAD_WARP_TRANSPOSE,
-                                    MayAlias ? LOAD_CA : LOAD_LDG,
-                                    BLOCK_STORE_WARP_TRANSPOSE>;
-  };
-
-  using MaxPolicy = Policy350;
+  struct Policy350
+      : cub::detail::ptx_base<350>
```

Comment:
Is `_base` necessary? Can it be `ptx<350>` instead?

Reply:
I think so. The `_base` suffix is a leftover from when I had both `ptx_base` (intrusive) and `ptx_tag` (non-intrusive) versions of this functionality. The `tag` never got used, so it'd make sense to use the shorter name now.